src/GroupManager.php

```diff
@@ -77,12 +77,12 @@ public function getGroupsForEntityType($entity_type_id) {
    *
    * @return array
    */
-  public function getAllGroups() {
+  public function GetAllGroupBundles($entity_type = NULL) {
     if (!isset($this->groupMap)) {
       $this->refreshGroupMap();
     }
 
-    return $this->groupMap;
+    return !empty($this->groupMap[$entity_type]) ? $this->groupMap[$entity_type] : $this->groupMap;
```

Comment:
If no entity passed it should return all bundles keyed, by entity - that are groups